Traceback (most recent call last):
  File "C:\Users\khthe\AppData\Local\Programs\Python\envs\py311\Lib\site-packages\jupyter_cache\executors\utils.py", line 58, in single_nb_execution
    executenb(
  File "C:\Users\khthe\AppData\Local\Programs\Python\envs\py311\Lib\site-packages\nbclient\client.py", line 1265, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\khthe\AppData\Local\Programs\Python\envs\py311\Lib\site-packages\jupyter_core\utils\__init__.py", line 166, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\khthe\AppData\Local\Programs\Python\Python311\Lib\asyncio\base_events.py", line 653, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Users\khthe\AppData\Local\Programs\Python\envs\py311\Lib\site-packages\nbclient\client.py", line 703, in async_execute
    await self.async_execute_cell(
  File "C:\Users\khthe\AppData\Local\Programs\Python\envs\py311\Lib\site-packages\nbclient\client.py", line 1021, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\khthe\AppData\Local\Programs\Python\envs\py311\Lib\site-packages\nbclient\client.py", line 915, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# SO NICHT:
def ggt():
    a = int(input('Bitte eine positive ganze Zahl eingeben: '))
    b = int(input('Bitte noch eine positive ganze Zahl eingeben: '))
    while a != b:
        if a > b:
            a = a - b
        else:
            b = b - a

    print(f'Der grÃ¶ÃŸte gemeinsame Teiler beider Zahlen ist {a}.')
    
ggt()
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mStdinNotImplementedError[0m                  Traceback (most recent call last)
Cell [1;32mIn[16], line 13[0m
[0;32m      9[0m             b [38;5;241m=[39m b [38;5;241m-[39m a
[0;32m     11[0m     [38;5;28mprint[39m([38;5;124mf[39m[38;5;124m'[39m[38;5;124mDer grÃ¶ÃŸte gemeinsame Teiler beider Zahlen ist [39m[38;5;132;01m{[39;00ma[38;5;132;01m}[39;00m[38;5;124m.[39m[38;5;124m'[39m)
[1;32m---> 13[0m [43mggt[49m[43m([49m[43m)[49m

Cell [1;32mIn[16], line 3[0m, in [0;36mggt[1;34m()[0m
[0;32m      2[0m [38;5;28;01mdef[39;00m [38;5;21mggt[39m():
[1;32m----> 3[0m     a [38;5;241m=[39m [38;5;28mint[39m([38;5;28;43minput[39;49m[43m([49m[38;5;124;43m'[39;49m[38;5;124;43mBitte eine positive ganze Zahl eingeben: [39;49m[38;5;124;43m'[39;49m[43m)[49m)
[0;32m      4[0m     b [38;5;241m=[39m [38;5;28mint[39m([38;5;28minput[39m([38;5;124m'[39m[38;5;124mBitte noch eine positive ganze Zahl eingeben: [39m[38;5;124m'[39m))
[0;32m      5[0m     [38;5;28;01mwhile[39;00m a [38;5;241m!=[39m b:

File [1;32m~\AppData\Local\Programs\Python\envs\py311\Lib\site-packages\ipykernel\kernelbase.py:1190[0m, in [0;36mKernel.raw_input[1;34m(self, prompt)[0m
[0;32m   1188[0m [38;5;28;01mif[39;00m [38;5;129;01mnot[39;00m [38;5;28mself[39m[38;5;241m.[39m_allow_stdin:
[0;32m   1189[0m     msg [38;5;241m=[39m [38;5;124m"[39m[38;5;124mraw_input was called, but this frontend does not support input requests.[39m[38;5;124m"[39m
[1;32m-> 1190[0m     [38;5;28;01mraise[39;00m StdinNotImplementedError(msg)
[0;32m   1191[0m [38;5;28;01mreturn[39;00m [38;5;28mself[39m[38;5;241m.[39m_input_request(
[0;32m   1192[0m     [38;5;28mstr[39m(prompt),
[0;32m   1193[0m     [38;5;28mself[39m[38;5;241m.[39m_parent_ident[[38;5;124m"[39m[38;5;124mshell[39m[38;5;124m"[39m],
[0;32m   1194[0m     [38;5;28mself[39m[38;5;241m.[39mget_parent([38;5;124m"[39m[38;5;124mshell[39m[38;5;124m"[39m),
[0;32m   1195[0m     password[38;5;241m=[39m[38;5;28;01mFalse[39;00m,
[0;32m   1196[0m )

[1;31mStdinNotImplementedError[0m: raw_input was called, but this frontend does not support input requests.
StdinNotImplementedError: raw_input was called, but this frontend does not support input requests.

